README.md

```diff
@@ -50,9 +50,10 @@ In this file we'll specify some settings for our HTTP-calls.
 ```ocaml
 module Config = {
   let baseUrl = "https://your_graphql_api_endpoint.com/";
+  let headers = [];
 };
 
-include ReveryGraphqlHooks.Make(Config);
+module Graphql = ReveryGraphqlHooks.Make(Config);
```

Comment:
Hmm, as we're still in the `Graphql.re`-file, I think this still stands? Perhaps we can make it clearer!

```suggestion
include ReveryGraphqlHooks.Make(Config)
```

Reply:
Oh, I wrote all of that in one file. Should have read all of it :D